pages/linux/xterm.md

```diff
@@ -0,0 +1,23 @@
+# xterm
+
+> A terminal emulator for the X Window System.
+
+- Open the terminal with a title of `Example`:
+
+`xterm -T {{Example}}`
+
+- Open the terminal in fullscreen mode:
+
+`xterm -fullscreen`
+
+- Open the terminal with a dark blue background and yellow foreground (font color):
+
+`xterm -bg {{darkblue}} -fg {{yellow}}`
+
+- Open the terminal in screen position x=200 pixels and y=20 pixels, with 100 characters per line and 35 lines:
```

Comment:
I'd mention the position coordinates _after_ the width/height of the terminal, to match the order in which these parameters appear in the actual command below.

Reply:
Makes sense, updating it :+1: